### PR TITLE
Add tools for genome browsers to rnaseq docker

### PR DESCRIPTION
**Pull request by JenkoB**
## Checklist

* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [ ] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [ ] All inputs are used in process.
* [ ] All output fields have their ``re-save`` calls.

### CODE REVIEW

**Comment by mstajdohar** [comment]
Rebase

File: resolwe_bio/docker_images/rnaseq/packages-manual/bedtobigbed.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    kent \
+    bedToBigBed \
+    kent-v365 \
+    a66027e1a51f803fef6bf234b90d0e4d6deee6027acfe2c0a65712f3157cca5d \
+    https://drive.google.com/uc?id=1N2pUaJuHgCUS4gztTcoybrRWw1m8WNtv#.elf
+
+wget https://raw.githubusercontent.com/ENCODE-DCC/kentUtils/master/src/hg/lib/encode/narrowPeak.as
```
Comment (JureZmrzlikar) [inline resolwe_bio/docker_images/rnaseq/packages-manual/bedtobigbed.sh]: Please explain why do we first pull something from google drive and then also perform wget. Are there two programs to be installed? If there are, they should be put in separate commits...

Comment (JenkoB) [reply resolwe_bio/docker_images/rnaseq/packages-manual/bedtobigbed.sh]: This is a short file that is needed by this program in some cases. For this case is actually redundant (it was important for chipseq), so I will delete it. Thanks

**Comment by JenkoB** [comment]
@JureZmrzlikar @mzganec please review.
